misk/src/main/kotlin/misk/web/DashboardTab.kt

```diff
@@ -1,46 +1,78 @@
 package misk.web
 
 import misk.security.authz.AccessAnnotationEntry
+import misk.web.ValidWebEntry.Companion.slugify
 import javax.inject.Inject
 import javax.inject.Provider
 import kotlin.reflect.KClass
 
+/**
+ * A [WebTab] with additional fields to bind to a specific DashboardId, with a menu name and category
+ */
 class DashboardTab(
   slug: String,
   url_path_prefix: String,
+  val dashboardSlug: String,
```

Comment:
the distinction of slug to dashboardSlug is confusing to me as a reader. Is slug the slug of the dashboard tab, and dashboardSlug the slug of its corresponding dashboard?